packages/browser/src/browser/standalone.ts

```diff
@@ -83,24 +83,28 @@ if (globalAnalyticsKey) {
 }
 
 if (shouldPolyfill()) {
-  // load polyfills in order to get AJS to work with old browsers
-  const script = document.createElement('script')
-  script.setAttribute(
-    'src',
-    'https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.7.0/polyfill.min.js'
-  )
+  const loadScript = (url: string, onLoad: () => void) => {
+    const script = document.createElement('script')
+    script.setAttribute('src', url)
+    script.onload = onLoad
 
-  if (document.readyState === 'loading') {
-    document.addEventListener('DOMContentLoaded', () =>
+    if (document.readyState === 'loading') {
+      document.addEventListener('DOMContentLoaded', () =>
+        document.body.appendChild(script)
+      )
+    } else {
       document.body.appendChild(script)
-    )
-  } else {
-    document.body.appendChild(script)
-  }
-
-  script.onload = function (): void {
-    attempt(install)
+    }
   }
+  // load polyfills in order to get AJS to work with old browsers
+  loadScript(
+    'https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.7.0/polyfill.min.js',
```

Comment:
I think we should add allSettled in generic utils.
Extending this polyfill code has a couple issues:
1. potential CSP issues with jsdeliver
2. 
We should add it in code, the shouldPolyfill thing is not reliable -- look at the support table (allSettled was only added in FF 71, but we only polyfill below 46)
![image](https://github.com/segmentio/analytics-next/assets/5115498/487aca63-78b3-42d5-85e2-a10e452e3883)

The entire polyfill pattern that was designed here seems very hacky/fragile/manual and I am in favor of re-examining it (I even wrote tickets around it)